### DISCUSSION TITLE
Failed to combine multiple plt files (PeleAnalysis/Src/combine)

### DISCUSSION THREAD

**Discussion by hzw-zju-2023** [AMReX-Combustion/PeleC#784] [Q&A]
I tried to use some modules in PeleAnalysis for post-processing, and then used function combine to combine multiple plt files, but I found that there were partial errors in the combine field. After careful inspection, I was sure that there was no problem in my post-processing process, and the problem appeared in the step of combine plt files. I would like to ask what caused this problem. I have attached the image of my OH gradient. In addition, I would like to explain that my simulation is spray jet combustion, and my grid adopts 3-level adaptive fine, with a total mesh size of about 40 million. 
![Uploading 图片1.png…]()

**Comment by hzw-zju-2023** [comment]
![图片1](https://github.com/AMReX-Combustion/PeleC/assets/154048455/a33197e3-3cf6-4003-aaea-78c608c332a6)

**Reply by marchdf** [reply]
Thanks for reaching out! Can you post the steps you made (command line input) to use the combined functionality? @drummerdoc might know what going on.

**Reply by drummerdoc** [reply]
That is bizarre indeed!  I wonder if there is an assumption about all the levels have the same box arrays in the files being combined that is being violated.  Can you "combine" the same field from two plot files that are the same plot file and get the same issue?  (that is, the left and right plot files on input are the same file, and the comp chosen in the same so that you should get two copies of the same field).  Also, if you flip the left and right plot files on the command line, does it do the same as above?

**Reply by hzw-zju-2023** [reply]
Thank you for your reply. I have tested both of your proposed methods, but I find that the problem still exists. Things are a little strange, my colleagues around the use of this function will not have problems, of course, our cases are different

**Reply by drummerdoc** [reply]
Hmmm.  If you "diff" the Header files in both plot files (using "tail" to chop out the list of variables because those will be different), do you see any obvious indicators that the plot files may not be compatible?  Can you do the same to the Cell_H files down in the Level_? folders?  This is very weird.
